commands/System/Admin/update.js

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2017-2018 dirigeants. All rights reserved. MIT license.
```

Comment:
Can you up the year, please?
```suggestion
// Copyright (c) 2017-2019 dirigeants. All rights reserved. MIT license.
```